packages/perseus/src/widgets/grapher/grapher.tsx

```diff
@@ -250,10 +248,12 @@ class FunctionGrapher extends React.Component<FunctionGrapherProps> {
                             ) {
                                 const extraConstraint =
                                     this.props.model.extraCoordConstraint;
-                                // Calculat resulting coords and verify that
+                                // Calculate resulting coords and verify that
                                 // they're valid for this graph
-                                const proposedCoords = _.clone(this._coords());
-                                const oldCoord = _.clone(proposedCoords[i]);
+                                const proposedCoords = deepClone(
```

Comment:
Worried about my switch to `deepClone` from `_.clone` because `_.clone` is shallow. However the tests didn't pass when I switched to `{...this._coords()}` and did pass when I used `deepClone`.